src/services/BookService.tsx

```diff
@@ -1,16 +1,22 @@
-import axios from 'axios'
+import axios, { AxiosResponse } from 'axios'
 
 import TokenService from './TokenService'
+import BookRequest from '../model/BookRequest'
+import BookResponse from '../model/BookResponse'
 
 const token = TokenService.getAccesToken()
 
-const getAllBooks = () => {
+
+const urlParams = (bookRequest: BookRequest) => {
+  return `?PageNumber=${bookRequest.PageNumber.toString()}&PageLength=${bookRequest.PageLength.toString()}`
+}
+const getAllBooks = ( bookRequest : BookRequest) : Promise<AxiosResponse<BookResponse>> => {
   const config = {
     headers: {
```

Comment:
Again config is appearing in each function.

Reply:
We talked about this yesterday?